MSVC warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)

Warning ``inflate.c(1356,9): warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)`` introduced in commit 3eab3173ac7d1d53457452f3cd1eaeea5b2d43df

This warning is present in CI output:
https://github.com/zlib-ng/zlib-ng/actions/runs/3686988352/jobs/6239982515#step:12:37

cc @iii-i

--- a/inflate.c
+++ b/inflate.c
@@ -1352,7 +1352,7 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
     }
     copy->next = copy->codes + (state->next - state->codes);
     if (window != NULL) {
-        ZCOPY_WINDOW(window, state->window, 1U << state->wbits);
+        ZCOPY_WINDOW(window, state->window, (size_t)1U << state->wbits);
     }
     copy->window = window;
     dest->state = (struct internal_state *)copy;

